ush/machine/jet.yaml

```diff
@@ -40,3 +40,47 @@ data:
       netcdf: /public/data/grids/gfs/anl/netcdf
     RAP: /public/data/grids/rap/full/wrfprs/grib2
     HRRR: /public/data/grids/hrrr/conus/wrfprs/grib2
+
+# RRFS input obs./data
+RAP:
+  local:
+    file_path: /public/data/grids/rap/obs
+    file_names:
+      obs:
+        - "{yyyymmddhh}.rap.t{hh}z.prepbufr.tm00"
+        - "{yyyymmddhh}.rap.t{hh}z.1bamua.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.1bhrs4.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.1bmhs.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.amsr2.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.ascatt.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.ascatw.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.atms.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.atmsdb.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.crisf4.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.crsfdb.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.esamua.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.esatms.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.eshrs3.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.esiasi.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.esmhs.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.gpsipw.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.gpsipw.tm00.bufr_d.nr"
+        - "{yyyymmddhh}.rap.t{hh}z.gsrasr.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.gsrcsr.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.iasidb.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.lghtng.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.lghtng.tm00.bufr_d.nr"
+        - "{yyyymmddhh}.rap.t{hh}z.lgycld.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.mtiasi.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.nexrad.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.rassda.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.satwnd.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.sevasr.tm00.bufr_d"
+        - "{yyyymmddhh}.rap.t{hh}z.ssmisu.tm00.bufr_d"
+FV3GFS:
+  local:
+    file_path: 
+      - ['/public/data/grids/gfs/prepbufr', '/public/data/grids/gfs/bufr']
+    file_names:
+      obs:
+        - ['{yyjjjhh}00.gfs.t{hh}z.prepbufr.nr', '230081200.gfs.t12z.syndata.tcvitals.tm00']
```

Comment:
good suggestion of spitting the entries into gfs_prepbufr and gfs_tcvitals. and you are right, the template is needed for the gfs_vitals files; i hard-coded one in because at the time of this PR, only one file (230081200.gfs.t12z.syndata.tcvitals.tm00) existed at /public/data/grids/gfs/bufr, so i wasn't sure what the temporal frequency of the file dumps / retention timescale was going to be. will switch over to {yyjjjhh}.

Reply:
removed this combined fv3gfs section from the machine file. 

added new entry under "data" with fv3gfs_prepbufr path on jet (which gets file names from hpss entry of corresponding files in data_locations.yaml). i cannot find the gfs_tcvitals files on hpss, so i left that out of this for now.

Reply:
The syndat files, if available, should be updated regularly. That file, I'm pretty sure, should have a template in it. 

For the other suggestion, I was thinking something more like this:

```
data:
  ics_lbcs:
    FV3GFS: ....
    ....
  obs:
    RAP: /public/data/grids/rap/obs
    GFS:
      prepbufr: /public/data/grids/gfs/prepbufr
      tcvitals: /public/data/grids/gfs/bufr
        
```

This becomes a bit more readable for which ones are observations and which ones should be used for ics/lbcs. There won't be overlap there.

Reply:
i tried to match this suggestion as closely as possible, with the caveat that RAP and GFS are each appended by _obs (so that RAP is differentiated from the RAP section in data_locations.yaml, and in turn, GFS_obs to be consistent with RAP_obs)

Reply:
i.e.: 

obs:
    RAP_obs: /public/data/grids/rap/obs
    GFS_obs:
      prepbufr: /public/data/grids/gfs/prepbufr
      tcvitals: /public/data/grids/gfs/bufr